### DISCUSSION TITLE
スパムページの対応

### DISCUSSION THREAD

**Discussion by tmura00** [increments/qiita-discussions#286] [ヘルプ]
数ヶ月前にサポートにメールを送信し
御社からは規約に違反するため何かしらのご対応をいただけるとの事でしたが、
数ヶ月待っても状況が変わらないためこちらに投稿します。

Qiitaさんのサイトに記事の目次だけ書き、内容は全く無く、
ページ内に「**この記事は[こちらに移転しました。**」と自サイトへの誘導を行うページがございます。

googleで検索し上記ユーザのページが表示されますが、利便性が低いため困ります。

こちらからの質問は2点です。
1.このようなやり方は許させるのでしょうか？
2.御社からのレスポンスはございますが、規約違反の場合も何も対応できないのであればヤリ得ではないでしょうか？

**Comment by tanaka-asami415** [comment]
@tmura00 
ご不便をおかけし大変申し訳ございません。

記事の内容と直接的に関係ないURLに誘導したり、
記事上で完結させることも出来る情報を敢えて載せずに、外部のサイトに誘導することは
利用規約違反となりますので順次対応を行っております。

対応しきれていない部分があり、ご不便をおかけして申し訳ありません。
利用規約やコミュニティガイドライン違反の報告につきましては、引き続き通報機能をご利用いただけますと幸いです。

https://github.com/increments/qiita-discussions/discussions/271 のコメントへの回答もこちらに代えさせていただきます。
また、特定のアカウントをご指摘される場合は、サポートフォームをご利用ください。
#271のコメントは、恐れ入りますが削除させていただきます🙇

**Reply by tmura00** [reply]
@tanaka-asami415 
ご回答ありがとうございます。

> 利用規約やコミュニティガイドライン違反の報告につきましては、引き続き通報機能をご利用いただけますと幸いです。

残念ながら通報もサポートへのメールも過去に行いました。
サポートからは規約違反であるとメールをいただき対応する旨のご返信をいただきましたが、
数ヶ月以上お待ちしても現状が変わらないためこちらにアカウントを記載しました。

御社が手が回らないとの事ですが、数年放置している状況はやらないに等しい認識ですし、
削除されたコメントのユーザはgoogleで上位に検索表示されたページから自ページへ誘導されています。
こちらとしましては、御社のSEO対策の一貫としてPVを稼ぐ施策の一部ではないかと邪推してしまいます。
@tanaka-asami415 様お一人では難しい部分もございますが、
インターネットでご商売される以上機能の充実も重要ですが、
健全な運用は最優先の課題ではないでしょうか？

**Reply by tanaka-asami415** [reply]
@tmura00 
対応しきれておらず、ご不便をおかけして申し訳ありません。
今後はより対応を強化していきます。
